src/applications/simple-forms/mock-simple-forms-patterns/arrayBuilder/components/arrayBuilder.jsx

```diff
@@ -15,26 +15,7 @@ import ArrayBuilderCards from './ArrayBuilderCards';
 /**
  * @typedef {Object} ArrayBuilderPages
  * @property {function(FormConfigPage): FormConfigPage} summaryPage Summary page which includes Cards with edit/remove, and the Yes/No field
- * @property {function(FormConfigPage): FormConfigPage} [itemFirstPage] First page for the item
- * @property {function(FormConfigPage): FormConfigPage} [itemMiddlePage] A page between the first and last page for the item
- * @property {function(FormConfigPage): FormConfigPage} [itemLastPage] Last page for the item that will navigate back to the summary page
- * @property {function(FormConfigPage): FormConfigPage} [itemSinglePage] Use this if there's only one page
- */
-
-/**
- * @typedef {Object} ArrayBuilderCards
- * @property {(item) => string | JSX.Element} [description]
- * @property {(item) => boolean} [isItemIncomplete]
- */
-
-/**
- * @typedef {Object} ArrayBuilderCancel
- * @property {string} [title]
- */
-
-/**
- * @typedef {Object} ArrayBuilderRemove
- * @property {string} [title]
+ * @property {function(FormConfigPage): FormConfigPage} [itemPage] A repeated page corresponding to an item
```

Comment:
this is one of the main refactor changes, just having `itemPage` for any page instead of 4 different categories of pages.